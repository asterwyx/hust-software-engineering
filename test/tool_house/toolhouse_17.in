preset user AQ
preset credit A 1000
preset credit Q 40
preset gift Q robot 10
preset userloc A 26 0
preset userloc Q 25 0
preset nextuser Q
step 3
step 1
n
dump