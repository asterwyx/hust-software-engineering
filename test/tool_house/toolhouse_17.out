user AQ
fund A 10000
credit A 1000
userloc A 27 0
fund Q 10000
credit Q 40
userloc Q 28 0
gift Q robot 10
nextuser Q